### ISSUE TITLE
Unable to connect to qemu monitor (port 5000), retrying in a second

### ISSUE THREAD

**Issue by Konstigt** [hellt/vrnetlab#102]
I cannot get this to work. The SROS version used is not listed as supported but my colleague has it up and running.. both Ubuntu 20.04 and 22.04 (I've tried both). I have containerlab 0.38.0 and a matching vrnetlab is not listed in the [compabitlity matrix](https://containerlab.dev/manual/vrnetlab/#compatibility-matrix). But I have tried just git clone and also with "git checkout v0.9.0" which was the latest listed.

It complains in docker about modules.. but the modules exists and are loaded (I can run both ip6tables -t nat -L -n -v and ip6tables  -L -n -v). 

Attaching full output from "docker logs  clab-vr-sros-r1"

`2023-03-15 19:18:19,029: vrnetlab   INFO     Unable to connect to qemu monitor (port 5000), retrying in a second (attempt 60)
Traceback (most recent call last):
  File "/launch.py", line 1059, in <module>
    ia.start(add_fwd_rules=False)
  File "/vrnetlab.py", line 711, in start
    vm.work()
  File "/vrnetlab.py", line 642, in work
    self.check_qemu()
  File "/vrnetlab.py", line 656, in check_qemu
    self.start()
  File "/vrnetlab.py", line 208, in start
    raise QemuBroken(
vrnetlab.QemuBroken: Unable to connect to qemu monitor on port 5000`

[logs.txt](https://github.com/hellt/vrnetlab/files/10983977/logs.txt)
[ip.txt](https://github.com/hellt/vrnetlab/files/10983992/ip.txt)

**Comment by hellt** [comment]
your system can't start qemu for whatever reason, so the issue is not (yet) with the iptables modules.
Maybe not enough memory? Nested virt not enabled?

**Comment by Konstigt** [comment]
The Ubuntu host had 6GB of memory but I increased it now to 12GB. No changes there. Nested should be OK!

root@clab:/etc/containerlab# cat /sys/module/kvm_intel/parameters/nested
Y
root@clab:/etc/containerlab# free
               total        used        free      shared  buff/cache   available
Mem:        12257252      402676    10091872        1328     1762704    11582820
Swap:        1984508           0     1984508

**Comment by hellt** [comment]
Unfortunately there is nothing to recommend but checking your environment.
You can try to launching sros with qemu directly, without containerlab, all points out that qemu failed to start for whatever reason.
These reasons are most likely related to your infra.

Because there is nothing here we can do from vrnetlab/containerlab perspective I am closing this.
Feel free to update the issue if you figure out what was with your environment.

**Comment by Konstigt** [comment]
I'm pretty sure it was the memory after all. I got the same problem with 12G.. I thought! But checking again later it connected after 12s (so the same error but not until failure).

I added even more memory and also added BOOT_DELAY and now they are all up and running without any other changes.

root@clab:/etc/containerlab# docker ps --format '{{.Names}} {{.Status}}'
clab-vr-sros-r6 Up 24 minutes (healthy)
clab-vr-sros-r3 Up 24 minutes (healthy)
clab-vr-sros-r4 Up 24 minutes (healthy)
clab-vr-sros-r2 Up 24 minutes (healthy)
clab-vr-sros-r5 Up 24 minutes (healthy)
clab-vr-sros-r1 Up 24 minutes (healthy)
